marklogic-data-hub/src/test/java/com/marklogic/hub/flow/impl/FlowRunnerTest.java

```diff
@@ -81,6 +83,32 @@ public void testRunFlow() {
         Assertions.assertTrue(file.contains("ingest.csv"));
     }
 
+    @Test
+    void runFlowWithStemmingSearch(){
+        try{
+            updateStemmingOnStagingDb(true);
+            testRunFlow();
+        }
+        finally {
+            updateStemmingOnStagingDb(false);
+        }
+
+    }
+
+    private void updateStemmingOnStagingDb(boolean enableStemming) {
+        HubConfig adminConfig = runAsDataHubDeveloper();
```

Comment:
Just call this "hubConfig", as "adminConfig" implies it's an admin user.

Reply:
Initially, I thought I needed to get runAsAdmin(), so had it named 'adminConfig' but later realized I could do it as developer, but forgot to rename it. Will do